src/sidebar/components/selection-tabs.js

```diff
@@ -87,7 +85,7 @@ function SelectionTabs({ isLoading, settings }) {
 
   const isThemeClean = settings.theme === 'clean';
 
-  const selectTab = function(type) {
+  const selectTab = type => {
```

Comment:
Might we consider another name for this parameter, for clarity? `type` doesn't feel quite right to me.